main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -893,6 +895,15 @@ private void dropCache() {
     }
 
     private void storeCache() {
```

Comment:
Are there many call sites? If not, just call `storeCache(false)` there instead of having this tiny private proxy method.